src/main/java/com/soen343/gms/service/JobServiceImplementation.java

```diff
@@ -0,0 +1,31 @@
+package com.soen343.gms.service;
+
+import com.soen343.gms.model.Job;
+import com.soen343.gms.model.Task;
+import com.soen343.gms.repository.JobRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service("jobService")
+public class JobServiceImplementation implements JobService {
+    @Autowired
+    JobRepository jobRepository;
+
+    @Override
+    public void saveJob(Job job) {
+        jobRepository.save(job);
+    }
+
+    @Override
+    public List<Job> getAllJobs() {
+        return jobRepository.findAll();
+    }
+
+    @Override
+    public Job getJobById(long id) {
+        return jobRepository.getOne(id);
```

Comment:
The standard for the framework would be to use findJobById(id) method name for a READ(find) operation. Since we are using a framework for CRUD operations we should try not do deviate from it.